subsys/bluetooth/services/fast_pair/fp_storage/fp_storage_ak.c

```diff
@@ -488,13 +1076,45 @@ static int fp_storage_ak_reset(void)
 		}
 	}
 
+	if (IS_ENABLED(CONFIG_BT_FAST_PAIR_STORAGE_AK_BOND)) {
+		for (uint8_t index = 0; index < ARRAY_SIZE(fp_bonds); index++) {
+			if (bt_addr_le_eq(&fp_bonds[index].nv.addr, BT_ADDR_LE_ANY)) {
+				continue;
+			}
+
+			err = bt_request_cb->bond_remove(&fp_bonds[index].nv.addr);
```

Comment:
Consider reusing `bond_remove_completely`/`bond_remove`

Reply:
I'd leave it as is. We clear the settings and RAM explicitly during reset anyway. The callback may not appear if the Fast Pair is disabled so we can't wait for it.